lib/vf_analyzers.cpp

```diff
@@ -627,7 +627,7 @@ struct ValueFlowAnalyzer : Analyzer {
         if (invalid())
             return Action::Invalid;
         // Follow references
-        auto refs = followAllReferences(tok);
+        auto refs = tok->refs();
```

Comment:
This copy is necessary since an additional entry is being added. But I think this is not necessary and I will try to refactor the code to avoid it.

Reply:
I adjusted this by unfortunately there is some redundant code introduced.